static/src/js/components/PlatformField/PlatformField.jsx

```diff
@@ -0,0 +1,259 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import Select from 'react-select'
+import fetchCmrKeywords from '../../utils/fetchCmrKeywords'
+import parseCmrResponse from '../../utils/parseCmrResponse'
+import './PlatformField.scss'
+
+/** This is custom field found with Collections >> acquisitionInformation
+ *
+ * @typedef {Object} PlatformField
+ * @property {Function} onChange A callback function triggered when the user inputs a text.
+ * @property {Object} uiSchema uiSchema for the field being shown.
+ * @property {Object} formData Saved Draft
+ */
+const PlatformField = (props) => {
```

Comment:
Pull out `onChange`, `uiSchema` and `formData` from the the props and only pass those values into the component.

Look at PreviewMapTemplate as an example